### DISCUSSION TITLE
Integrating  Azure DevOps (formerly VSTS) API to API execution via artillery

### DISCUSSION THREAD

**Discussion by ChetanKB** [artilleryio/artillery#1619] [Artillery]
how to add the Azure DevOps (formerly VSTS) API to the Artillery  API test runner image so that API test results can be seen In azure pipelines.

**Comment by vikramjb** [comment]
@ChetanKB It's covered in the below page from artillery's documentation

https://www.artillery.io/docs/cicd/azure-devops
